python/taichi/lang/meta.py

```diff
@@ -16,13 +16,24 @@ def tensor_to_ext_arr(tensor: ti.template(), arr: ti.ext_arr()):
 
 
 @ti.kernel
-def vector_to_fast_image(img: ti.template(), out: ti.ext_arr()):
+def vector_to_fast_image(img: ti.template(), small_endian: ti.template(),
+                         out: ti.ext_arr()):
     # FIXME: Why is ``for i, j in img:`` slower than:
     for i, j in ti.ndrange(*img.shape):
         u, v, w = min(255, max(0, int(img[i, img.shape[1] - 1 - j] * 255)))
```

Comment:
Possible my fault, use `r, g, b` instead of `u, v, w` for clarity?

Reply:
Yep, fixed